evm-contracts/src/v0.7/vendor/Address.sol

```diff
@@ -0,0 +1,190 @@
+// SPDX-License-Identifier: MIT
+// From https://github.com/OpenZeppelin/openzeppelin-contracts v4.1.0(23869e5b2a7c6b9c3e27dee4289615b8cf50e36b)
+
+pragma solidity ^0.7.0;
```

Comment:
This doesn't conform to our contract style guide.

I know this is imported from the OpenZeppelin library... so I'm not going to die on this hill. However, the rest of `vendor/` does conform.

Reply:
I think optimizing for audit is important, and changing style of vendored contracts could make it difficult to tell that nothing _really_ changed. I think we should leave everything as close as possible for vendored contracts.

As for the stuff that is already vendored but stylized, I'd go as far as to say we should go back and make sure they are as close as possible to the source.